docs/communications/examples.rst

```diff
@@ -4,24 +4,21 @@ Examples
 
 The following are examples of how the flow of messages will be for each mission. All fields will be provided.
 
-------------
-
-.. toctree::
-  :caption: Mission Examples
-  :maxdepth: 1
-
-  examples/isr-search
-
-------------
-
 .. note::
   All examples will be run independently. Pretend that each example is as if the mission was started first, and no vehicles were connected to the GCS.
 
   In real missions, the ``id`` from the GCS when the vehicle connects can be higher if the GCS has performed missions with previous vehicles.
 
-
 .. note::
-  All ``id`` fields will be filled in with an incrementing number. Every platform can implement this field differently; the only requirement is that the ``id`` field increases in value for every unique message sent. See `here <implementation.html#message-management>`_ for more information on how to generate these.
+  All ``id`` fields will be filled in with an incrementing number. Every platform can implement this field differently; the only requirement is that the ``id`` field increases in value for every unique message sent. See `here <implementation.html#message-management>`__ for more information on how to generate these.
```

Comment:
All instances of `` `hyperlink text <somelink.com>`_ `` are replaced with one of the following:

1. text with link at the bottom of the page

```rst
Some text with a `hyperlink text`_ in it

More text...

.. _some text: somelink.com
```

2. text with link together with an anonymous anchor (has two _ after it). This is because there is case where Sphinx reports errors if the anchor is not anonymous.

```rst
Some text with a `hyperlink text <somelink.com>`__ in it

More text...
```